lib/middleware/request-logger-middleware.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Assign a random ID to the request.
+ *
+ * Use this as middleware in your route files.
+ */
+
+"use strict";
+
+module.exports = (config, logger, random) => {
+    var idLength;
+
+
+    /**
+     * Log a message for the request.  Makes sure the log entry is
+     * prefixed with the log ID and a date/time stamp.
+     *
+     * @param {string} message
+     * @this {restify~Request}
+     */
+    function logMessage(message) {
+        var d;
+
+        d = new Date();
+        logger.info(`${this.logId} ${d.toISOString()} ${message}`);
+    }
+
+
+    idLength = config.server.requestIdLength;
+
+    /**
```

Comment:
I think another newline above the JSDoc and one less newline above `idLength = config.server.requestIdLength;`